custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
+        if self.child_index == 1:
+            form_prop = 'interpret_grade'
+        else:
+            form_prop = 'interpret_grade_{}'.format(self.child_index)
+
+        forms = self.filtered_forms(CHILDREN_FORMS, 1)
+        if len(forms) == 0:
+            return False
```

Comment:
Do you know if there should be some vhnd availability condition here?  If there are no matching forms, but there was also no VHND available, does the condition apply?